python/ray/train/session.py

```diff
@@ -41,7 +43,7 @@ class TrainingResult:
     data: Dict
 
 
-class Session:
+class TrainSession(Session):
```

Comment:
The same concern goes for TrainSession - Too many irrelevant APIs exposed to user.

Reply:
Resolved by explicitly calling out the public facing interface: TuneSession and TrainSession.